lib/ProgressPlugin.js

```diff
@@ -30,8 +31,8 @@ const createDefaultHandler = profile => {
 			}
 			for (let detail of details) {
```

Comment:
There can be multiple detail strings. It would be great to measure to total length first and then decide how much should be skipped from each.